enterprise/tpa_pipeline.py

```diff
@@ -62,11 +62,18 @@ def validate_provider_config(enterprise_customer, sso_provider_id):
     """
     Helper method to ensure that a customer's provider config is validated
     """
+    enterprise_orchestration_config = enterprise_customer.sso_orchestration_records.filter(
```

Comment:
excuse me if this is a naive question but will the TPA pipeline ever be invoked with these configs? isnt auht0 handling all that?